### PR TITLE
Implement DB methods for Landmarks

### LINKED ISSUE #131: Implement DB methods for Landmarks

### LINKED ISSUE #135: Implement DB methods for Following

### PR DESCRIPTION
**Pull request by kevinzhuang10**
closes #131 
closes #135

### CODE REVIEW

File: server/controllers/posts.js [hunk 1]
```diff
@@ -31,15 +37,24 @@ module.exports.getPostsByStoryId = (req, res) => {
...
+  models.Post.getPostsByLandmarkId(req.params.id)
     .then(results => {
       res.status(200).send(results);
     });
 };
 
-module.exports.getPostsByFollowing = (req, res) => {
-  models.Post.getPostsByFollowing(req.body)
+module.exports.getPostsByFollowings = (req, res) => {
+  models.Following.getAllFollowings(req.params.id)
```
Comment (lazdinst) [inline server/controllers/posts.js:47]: Is req.params.id the id of /api/posts/followings/:id?

**Review by lazdinst — APPROVED** [review]

File: server/controllers/posts.js (hunk 1 above)
Comment (kevinzhuang10) [reply server/controllers/posts.js:47]: Sorry, that's actually the current user id

File: client/src/components/main/CreateNewPost.jsx [hunk 2]
```diff
@@ -46,6 +46,7 @@ const CreateNewPost = (props) => {
     // listen for location selection from the dropdown
     google.maps.event.addListener(autocomplete, 'place_changed', () => {
       place = autocomplete.getPlace();
+      console.log(place);
```
Comment (Sam0064) [inline client/src/components/main/CreateNewPost.jsx:49]: Is this console.log necessary in our master branch??

Comment (kevinzhuang10) [reply client/src/components/main/CreateNewPost.jsx:49]: No it's not. But we do have a bunch of console logs in master currently

Comment (Sam0064) [reply client/src/components/main/CreateNewPost.jsx:49]: Do you mind taking it out?? It's just one more piece of code cleanup that we won't have to chase down later. Thanks.

Comment (lazdinst) [reply client/src/components/main/CreateNewPost.jsx:49]: Console logging data i think is unnecessary but console logging error/success status as actions execute to show the current state is ok.

**Pull request merged**